docs/en/Logging-Conventions.md

```diff
@@ -140,3 +140,12 @@ And the following command gets all workers' log level on class `alluxio.heartbea
 ```bash
 alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
 ```
+
+### Client-side Logging Configuration
+
+Often it's useful to change the logLevel of the Alluxio client running in the compute framework (e.g. Spark, Presto) process, and save it to a file for debugging. To achieve this, you can pass the following Java options to the compute 
+framework process.
+
+For example, the options `-Dalluxio.logs.dir=/var/alluxio/ -Dalluxio.logger.type=USER_LOGGER -Dlog4j.configuration=/tmp/
+alluxio/conf/log4j.properties` will instruct Alluxio client to use the log4j configuration in the Alluxio's conf path and 
```

Comment:
The conf path is some arbitrary path, that is on the client machine, right? What happens if the client does not have access to the server log4j conf files?

Reply:
That's right. If the client is not on the same node as the worker then a copy of that can be put on the client machine. I'll update the docs.